subiquity/curtin.py

```diff
@@ -25,7 +25,7 @@
 
 TMPDIR = '/tmp'
 CURTIN_SEARCH_PATH = ['/usr/local/curtin/bin', '/usr/bin']
-CURTIN_INSTALL_PATH = ['/media/root-ro', '/']
+CURTIN_INSTALL_PATH = ['/media/root-ro', '/rofs', '/']
```

Comment:
Not quite. This is just alternatives from which to grab the source data, with / being the last alternative (as it's always there). /media/root-ro does not normally exist, and /rofs is always the bottom-layer squashfs mounted by casper. Further alternatives may be added in the future, say if ubuntu-image mounts things differently.